.gitignore

```diff
@@ -1,10 +1,7 @@
 # Don’t commit the following directories created by pub.
-build/
-packages
-.buildlog
 .packages
-.pub
 .dart_tool
+pkg/**/pubspec.lock
```

Comment:
Please revert, we've decided to include the pubspec.lock files in all of these projects (caused CI failures to not track the dependencies).

Reply:
There were two things in `/pkg` that didn't have lock files.

Should I include those in this CL or leave it for later?

Reply:
Up to you, feel free to leave it later.